packages/integrations/image/README.md

```diff
@@ -102,7 +102,7 @@ For images in your project's repository, use the `src` relative to the `public`
 
 <p>
 
-**Type:** 'avif' | 'jpeg' | 'png' | 'webp'<br>
+**Type:** `avif` | `jpeg` | `png` | `webp`<br>
```

Comment:
Oh interesting, looking at the auto-gen for the Configuration Reference it looks like maybe this should be:

```
**Type:** `'avif' | 'jpeg' | 'png' | 'webp'`<br>
```

Not sure if that would render out exactly the same on the integration pages though

Reply:
Oh, that could be, too! I'm honestly not sure myself, but looking at the README just on GitHub, I can see it's the two sections using avif etc. where the rendering looks different.

Reply:
Yup, we should keep the quotes around each item (indicates a TypeScript string literal) here but wrap the whole union in backticks:

```suggestion
**Type:** `'avif' | 'jpeg' | 'png' | 'webp'`<br>
```